### PR TITLE
Zint Barcode Studio

### PR DESCRIPTION
**Pull request by frhun**
This packaging uses the currently latest tarball as a base, and adds
everything needed for the application to work as a flatpak, along with a
new hi-res icon.

The changes and additions made here would be better placed upstream in
the long term, but for now they make the flatpak work.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://sourceforge.net/p/zint/mailman/zint-barcode/thread/7dcb44bf-4ba1-16eb-88ad-e73e9fc905e9%40frhun.de/#msg37257326
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
 - The domain is owned by the original developer, and only associated with the project
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
 - (not through a formal PR, but mentioned in the mail to the Mailing List)

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build uk.org.zint.zint-qt

**Comment by frhun** [comment]
Everything seems to work fine. Although i haven't tested on arm yet, so that would be nice.

File: uk.org.zint.zint-qt.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  # this, and the following env is needed for that to work
+  # info: https://github.com/flathub/org.kde.okular/issues/41
+  # TODO: just mirror what KDE is doing here https://github.com/flathub/org.kde.kontact/blob/master/org.kde.kontact.json
+  - "--env=QT_QPA_PLATFORM=xcb"
+  - "--device=dri"
+  - "--filesystem=home:ro"
+  - "--filesystem=xdg-documents:rw"
+modules:
+  - name: zint-qt
+    buildsystem: cmake
```
Comment (barthalion) [inline uk.org.zint.zint-qt.yaml:21]:
```suggestion
    buildsystem: cmake-ninja
```

**Comment by barthalion** [comment]
bot, build uk.org.zint.zint-qt

**Comment by barthalion** [comment]
/merge

Let us know if someone from upstream team replies.

**Pull request closed without merging**